Tests/MongoMobileTests/MongoMobileTests.swift

```diff
@@ -1,19 +1,76 @@
-@testable import MongoMobile
 import XCTest
+import MongoSwift
+
+@testable import MongoMobile
 
 final class MongoMobileTests: XCTestCase {
-    static var allTests: [(String, (ClientTests) -> () throws -> Void)] {
+    static var allTests: [(String, (MongoMobileTests) -> () throws -> Void)] {
         return [
-            ("testMongoMobile", testMongoMobile)
+            ("testMongoMobileBasic", testMongoMobileBasic),
+            ("testSequentialAccess", testSequentialAccess)
         ]
     }
 
-    func testMongoMobile() throws {
-        let client = MongoMobile.create(settings: MongoClientSettings(dbPath: "test-path"))
+    // NOTE: These only works because we have one test suite. These method are called
+    //       before/after all tests _per_ test suite. Will not work if another suite
+    //       is added.
+    override class func setUp() {
+        super.setUp()
+        try? MongoMobile.initialize()
+    }
+
+    override class func tearDown() {
+        super.tearDown()
+        try? MongoMobile.close()
+    }
+
+    func createAndCleanTemporaryPath(at path: String) throws -> URL {
+        let supportPath = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask)[0]
+        let databasePath = supportPath.appendingPathComponent(path)
+
+        do {
```

Comment:
you made this method throw, why not propagate the errors?
alternatively, if you don't want to propagate them for some reason, make it non-throwing and swallow the errors with `try?` rather than do/catch.

Reply:
good catch, yeah I just wanted to try and ignore all thrown errors